actors/migration/nv8/market.go

```diff
@@ -0,0 +1,78 @@
+package nv7
```

Comment:
This migration needs to come with network version 9. Version 8 is already live. Please add a new package for nv9.